sentry/entrypoint.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -e
 
+if [ "$(ls -A /usr/local/share/ca-certificates/)" ]; then
+  update-ca-certificates
+fi
+
```

Comment:
Can we get [this logging](https://github.com/getsentry/sentry/pull/27062/commits/449412b925fbef18963f3b0bb95133e8caa82841) here to aid future debugging?

Reply:
`update-ca-certificats` _always_ outputs something, hence the former debugging lines were not necessary.

Reply:
I guess my argument would be that I don't know what `update-ca-certificates` outputs, and it's easier for me to find something grep for by looking in here than by learning more about `update-ca-certificates`. Not a blocker.